ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -147,19 +149,38 @@ export const getUpdatedProfileValue = (
   }
 };
 
-export const enableSave = (
+export const isBypassed = (
+  profile: Profile,
+  moduleName: AccessModule
+): boolean =>
+  !!getAccessModuleStatusByName(profile, moduleName)?.bypassEpochMillis;
+
+// would this AccessBypassRequest actually change the profile state?
+// allows un-toggling of bypass for a module
+export const wouldUpdateBypassState = (
+  oldProfile: Profile,
+  request: AccessBypassRequest
+): boolean => isBypassed(oldProfile, request.moduleName) !== request.isBypassed;
+
+export const isChanged = (
```

Comment:
note: new version does not check for the existence of errors.  this allows it to be used for more purposes besides "is the save button enabled"

Reply:
could you say more about that?

Reply:
Something little more verbose along the lines of `hasBypassChanged` or `isBypassChanged` to make it more obvious.

Reply:
ok - it's the profile which has changed here.  So it can "profileNeedsUpdate()" or something